app/exchanges/run_csv/pacbio_sample_sheet.rb

```diff
@@ -87,11 +87,12 @@ def smrt_cell_settings
     end
 
     def sample_data(well, sample)
+      adapter_suffix = sample.tag&.oligo_reverse.present? ? %w[_L _R] : ['', '']
```

Comment:
I am just wondering if it would be better to have methods in the tag model adapter_1 and adapter_2? They are then more easily testable and maintanable.